runtime/zeitgeist/src/integration_tests/xcm/tests/transfers.rs

```diff
@@ -172,6 +172,242 @@ fn transfer_ztg_sibling_to_zeitgeist() {
+                        Parachain(zeitgeist::ID),
+                        Junction::AccountId32 { network: NetworkId::Any, id: ALICE.into() }
+                    )
+                )
+                .into()
+            ),
+            Limited(4_000_000_000),
+        ));
+
+        // Confirm that Alice's balance is initial_balance - amount_transferred
+        assert_eq!(Balances::free_balance(&ALICE), sibling_alice_initial_balance - transfer_amount);
+
+        // Verify that the amount transferred is now part of the zeitgeist account here
+        assert_eq!(
+            Balances::free_balance(zeitgeist_parachain_account()),
+            initial_sovereign_balance + transfer_amount
+        );
+    });
+
+    Zeitgeist::execute_with(|| {
+        let expected = transfer_amount - btc_fee();
+        let expected_adjusted = adjusted_balance(btc(1), expected);
+
+        // Verify that remote Alice now has initial balance + amount transferred - fee
+        assert_eq!(
+            Tokens::free_balance(BTC_ID, &ALICE),
+            zeitgeist_alice_initial_balance + expected_adjusted,
+        );
+    });
+}
+
+#[test]
+fn transfer_btc_zeitgeist_to_sibling() {
+    TestNet::reset();
+
+    let transfer_amount = btc(100) - btc_fee();
+    let initial_sovereign_balance = 2 * btc(100);
+    let sibling_bob_initial_balance = btc(0);
+
+    transfer_btc_sibling_to_zeitgeist();
+
+    Sibling::execute_with(|| {
+        assert_eq!(Tokens::free_balance(BTC_ID, &BOB), sibling_bob_initial_balance,);
+    });
+
+    Zeitgeist::execute_with(|| {
+        assert_ok!(XTokens::transfer(
+            RuntimeOrigin::signed(ALICE),
+            BTC_ID,
+            transfer_amount,
+            Box::new(
+                MultiLocation::new(
+                    1,
+                    X2(
+                        Parachain(PARA_ID_SIBLING),
+                        Junction::AccountId32 { network: NetworkId::Any, id: BOB.into() }
+                    )
+                )
+                .into()
+            ),
+            Limited(4_000_000_000),
+        ));
+
+        // Confirm that Alice's balance is initial_balance - amount_transferred
+        assert_eq!(Tokens::free_balance(BTC_ID, &ALICE), 0);
+    });
+
+    Sibling::execute_with(|| {
+        let fee_adjusted = adjusted_balance(btc(1), btc_fee());
+        let expected = transfer_amount - fee_adjusted;
+
+        // Verify that Bob now has initial balance + amount transferred - fee
+        assert_eq!(Balances::free_balance(&BOB), sibling_bob_initial_balance + expected,);
+
+        // Verify that the amount transferred is now subtracted from the zeitgeist account at sibling
+        assert_eq!(
+            Balances::free_balance(zeitgeist_parachain_account()),
+            initial_sovereign_balance - transfer_amount
+        );
+    });
+}
+
+#[test]
+fn transfer_eth_sibling_to_zeitgeist() {
+    TestNet::reset();
+
+    let sibling_alice_initial_balance = ztg(10) + eth(1);
```

Comment:
What is the point of adding those two different amounts?

Reply:
The ztg is used to pay native fees (although Alice should already have 10 ZTG from Genesis).
`eth` is just used for transparency, in the end it is stored as native ZTG but treated like eth on the chain that receives the XCM.